src/index.js

```diff
@@ -46,7 +46,7 @@ history.listen((location) => {
   const locationPage = location.hash && location.hash[0] === '#' && location.hash.slice(1);
   ReactGA.set({ page: locationPage });
   ReactGA.pageview(locationPage);
-  // TODO: Also notify navigation to DataHub (DataHub does not have this feature yet)
+  _dha.track({ navigatedPage: locationPage }); // eslint-disable-line 
```

Comment:
What is the disabled rule?

Reply:
Maybe you could do something like this:

```js
   window._dha && window._dha.track({ navigatedPage: locationPage });
```

Reply:
To prevent eslint from throwing a warning, because `_dha` is a global variable not defined in `index.js`